test/setup.js

```diff
@@ -0,0 +1,5 @@
+require('babel-register')({
+  ignore: name => name.includes('node_modules') && !name.includes('obs-store'),
```

Comment:
By default, babel doesn't compile anything from `node_modules`. This means that the `ObservableStore` class provided by `obs-store` was never getting transpiled to ES5. Ideally, `obs-store` should provide a UMD module instead of an ES6 class, but to work around this, we just tell babel to process `ObservableStore` as if it were part of our application.

This issue only became apparent when I tried to extend `ObservableStore`. Just using `ObservableStore` doesn't show the issue since our supported platforms support native Classes. However, when our transpiled ES5 application code attempted to extend an non-transpiled ES6 class, things blew up.

Reply:
I've opened this suggestion as a new issue on `obs-store`:
https://github.com/kumavis/obs-store/issues/6